### DISCUSSION TITLE
login information won't save / won't sync

### DISCUSSION THREAD

**Discussion by nicholus-h** [Taxel/PlexTraktSync#1677] [Support] [answered]
Running through portainer.

Usually when I run login, I get through the Plex login, and then it errors once after establishing a connection to the server. I run login again, and I get through the Plex login, and then it connects and I get through the trakt login.

After that, whenever I run sync, it continues to prompt me for the Plex login information every time. It connects no problem, but then as it tries to sync, I get a bunch of errors, including "unathorized" errors with HTML. Then, it says "no more methods to connect, giving up" and closes.

I had gotten it working it once, and then changed some of the settings around. I reset the Portainer stack / docker compose and deleted the entire configuration folder linked in the compose, but it behaves the same way every time.

[plextraktsync.log](https://github.com/Taxel/PlexTraktSync/files/13776826/plextraktsync.log)

**Comment by glensc** [comment, marked as answer]
wrong report kind. all required information missing, that would issue template ask. so, you will not get support here.

https://github.com/Taxel/PlexTraktSync/issues/new/choose

<img width="1008" alt="image" src="https://github.com/Taxel/PlexTraktSync/assets/199095/c7284389-7d43-48f0-9cd7-5378c799c31f">
